src/Hpp.hs

```diff
@@ -111,7 +112,7 @@ emptyHppState = T.emptyHppState
 
 -- | Create a 'T.HppState' with the given 'C.Config' and 'T.Env'.
 initHppState :: C.Config -> T.Env -> T.HppState
-initHppState c e = T.HppState c 1 e
+initHppState c e = setL lineNum 1 . setL env e . setL config c $ emptyHppState
```

Comment:
Done to make things a bit more consistent. The `config` lens setter in particular changes the value of the `dir` as well.